trafficlight/config.py

```diff
@@ -18,6 +18,7 @@ class Config(BaseModel):
     port: int = 3335
     output: Output = Output.UI
     webhook: str = ""
+    use_golbat_raws: bool = True
```

Comment:
default to `False` pls... that's a breaking change otherwise

Reply:
Ok ok